content/hi/training/_index.html

```diff
@@ -0,0 +1,154 @@
+          <img src="/images/training/kubernetes-ckad-white.svg"/>
+        </div>
+        <div class="logo-certification cta-image" id="logo-cks">
+          <img src="/images/training/kubernetes-cks-white.svg"/>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="main-section padded">
+    <center>
+      <h2>edX पर निःशुल्क पाठ्यक्रम लें</h2>
+    </center>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>कुबेरनेट्स का परिचय <br> &nbsp;</b>
+        </h5>
+        <p>कुबेरनेट्स सीखना चाहते हैं? कंटेनरीकृत अनुप्रयोगों के प्रबंधन के लिए इस शक्तिशाली प्रणाली पर गहन प्राइमर प्राप्त करें।</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button" hreflang="en">कोर्स पर जाएँ</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>क्लाउड इंफ्रास्ट्रक्चर टेक्नोलॉजीज का परिचय</b>
+        </h5>
+        <p>ओपन सोर्स में अग्रणी, लिनक्स फाउंडेशन से सीधे क्लाउड प्रौद्योगिकियों के निर्माण और प्रबंधन की बुनियादी बातें सीखें।</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button"  hreflang="en">कोर्स पर जाएँ</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>लिनक्स का परिचय</b>
+        </h5>
+        <p>कभी लिनक्स नहीं सीखा? ताज़ा करना चाहते हैं? प्रमुख लिनक्स वितरण परिवारों में ग्राफिकल इंटरफ़ेस और कमांड लाइन दोनों का उपयोग करके लिनक्स का अच्छा कार्यसाधक ज्ञान विकसित करें।</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-linux" target="_blank" class="button" hreflang="en">कोर्स पर जाएँ</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div class="padded lighter-gray-bg">
+  <div class="main-section two-thirds-centered">
+    <center>
+      <h2>लिनक्स फाउंडेशन के साथ सीखें</h2>
+      <p>लिनक्स फाउंडेशन कुबेरनेट्स अनुप्रयोग विकास और संचालन जीवनचक्र के सभी पहलुओं के लिए प्रशिक्षक के नेतृत्व वाले और स्व-गति वाले पाठ्यक्रम प्रदान करता है।</p>
+      <br/><br/>
+      <a href="https://training.linuxfoundation.org/training/course-catalog/?_sft_technology=kubernetes" target="_blank" class="button">पाठ्यक्रम देखें</a>
+    </center>
+  </div>
+</div>
+
+<section id="get-certified">
+  <div class="main-section padded">
+    <h2>कुबेरनेट्स प्रमाणित प्राप्त करें</h2>
+    <div class="col-container">
+      <div class="col-nav">
+        <h5>
+          <b>कुबेरनेट्स और क्लाउड नेटिव एसोसिएट (KCNA)</b>
+        </h5>
+        <p>कुबेरनेट्स और क्लाउड नेटिव एसोसिएट (KCNA) परीक्षा कुबेरनेट्स और व्यापक क्लाउड नेटिव इकोसिस्टम में उपयोगकर्ता के मूलभूत ज्ञान और कौशल को प्रदर्शित करती है।</p>
+        <p>एक प्रमाणित KCNA संपूर्ण क्लाउड नेटिव इकोसिस्टम के वैचारिक ज्ञान की पुष्टि करेगा, विशेष रूप से कुबेरनेट्स पर ध्यान केंद्रित करते हुए।</p>
```

Comment:
Please recheck machine translations to ensure they sound and seem grammatically correct when read by a native speaker. A literal word-to-word translation is not expected.

```suggestion
        <p>KCNA प्रमाणित व्यक्ति संपूर्ण क्लाउड नेटिव इकोसिस्टम, और विशेष रूप से कुबेरनेट्स के वैचारिक ज्ञान की पुष्टि करेगा।</p>
```